k8s/helm-crd/templates/k8s.cloudogu.com_backupschedules.yaml

```diff
@@ -44,8 +44,8 @@ spec:
             status:
               description: BackupScheduleStatus defines the observed state of BackupSchedule
               properties:
-                currentKubectlImage:
-                  description: CurrentKubectlImage is the image currently used to create scheduled backups.
+                cronJobImage:
+                  description: CurrentCronJobImage is the image currently used to create scheduled backups.
```

Comment:
```suggestion
                  description: CronJobImage is the image currently used to create scheduled backups.
```

Reply:
Oh, seems I haven't regenerated the CRDs after changing that.